tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy
+envlist = py33,py34,py35,pypy3
 
 [testenv]
 deps=
 	-rtest_requirements.txt
```

Comment:
This file is out of date - There is no longer "test_requirements.txt" because we are using PBR and it uses `py.test` instead of `pytest`.
I think that instead of updating it, you should just delete it.